shared/menubar/files-container.desktop.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import * as FsTypes from '../constants/types/fs'
+import * as FsGen from '../actions/fs-gen'
+import {FilesPreview} from './files.desktop'
+import {remoteConnect, compose} from '../util/container'
+
+const mapStateToProps = (state) => ({
+  _tlfRows: [
+    {path: FsTypes.stringToPath('/keybase/team/zila.test/abc')},
+    {path: FsTypes.stringToPath('/keybase/team/zila.test/def')},
+  ],
```

Comment:
I know this is just placeholders, but it'd cause re-rendering whenever anything in the store state changes. So maybe move it to `mergeProps` for now?

Reply:
This is going to go away in the next PR, and I'm not going to merge this one before having that one approved.

Reply:
Also I don't think it'll cause re-rendering, since the shallow equal will stay the same.

Reply:
> Also I don't think it'll cause re-rendering, since the shallow equal will stay the same.

Why is it the same? `_tlfRows` is a new array every time this runs.

Reply:
Oh true. In any case I'll make sure we do it properly in the follow up. Thanks for calling it out!